dotcom-rendering/src/types/branding.ts

```diff
@@ -29,3 +30,26 @@ export interface EditionBranding {
 	};
 	branding?: Branding;
 }
+
+export type BrandingKind = BrandingType['name'];
+
+/**
+ * Branding that can be applied to an entire collection on a front
+ */
+export type CollectionBranding =
+	| {
+			kind: BrandingKind;
```

Comment:
I don't know if this is possible but could we be more specific in the naming here at all? What does kind describe? Is it who commissioned the brand? The brand owner maybe?

Reply:
I've inlined `BrandingType['name']` here, which is close to the [consumed by Frontend (https://github.com/guardian/commercial-shared/blob/35cdf4e1/src/main/scala/com/gu/commercial/branding/BrandingType.scala).

I agree `kind` isn't super clear here, but it's basically just to discriminate between the `"editorial"` case and the `BrandingType['name']` cases. I'm going to leave it for now (I've added some comments to help understanding), but will try and think of better naming for this.

Reply:
Cool!